edx_proctoring/callbacks.py

```diff
@@ -152,12 +155,17 @@ def get(self, request, attempt_code):  # pylint: disable=unused-argument
             )
 
         update_exam_attempt(attempt['id'], last_poll_timestamp=timestamp, last_poll_ipaddr=ip_address)
+        time_remaining_seconds = get_time_remaining_for_attempt(attempt)
```

Comment:
This was one of my initial concerns with this story. It stops at `0`, even before the exam begins, etc. We could change it to return `None` if it doesn't make sense that there's time remaining.